app/js/controllers/form-infos-complementaires.js

```diff
@@ -34,7 +34,14 @@ angular.module('ddsApp').controller('FormInfosComplementairesIndividusCtrl', fun
         }
     };
 
+    // $scope.submit = function() {
+    //     $state.go('infos_complementaires.adresse_contact', { droit: $stateParams.droit });
+    // };
+
     $scope.submit = function() {
-        $state.go('infos_complementaires.adresse_contact', { droit: $stateParams.droit });
+        SituationService.save($scope.situation).then(function() {
+            $scope.situation.infosComplementairesCaptured = true;
+            $state.go('download_cerfa', {droit: $stateParams.droit});
+        });
```

Comment:
There is only a success handler on this promise, if a failure occurs it will be completely silent. Either `$scope.submit` handles promises and you should `return` it, or add a failure handler, or use some built-in Angular promise method to signify you've ended the chain.
(see item 3 [here](http://pouchdb.com/2015/05/18/we-have-a-problem-with-promises.html) for additional refs)